src/java.base/macosx/classes/sun/nio/fs/BsdFileSystem.java

```diff
@@ -67,6 +70,38 @@ public Set<String> supportedFileAttributeViews() {
         return SupportedFileFileAttributeViewsHolder.supportedFileAttributeViews;
     }
 
+    /**
+     * Clones the file whose path name is {@code src} to that whose path
+     * name is {@code dst} using the {@code clonefile} system call.
+     *
+     * @param src the path of the source file
+     * @param dst the path of the destination file (clone)
+     * @param followLinks whether to follow links
+     *
+     * @return 0 on success, IOStatus.UNSUPPORTED_CASE if the call does not work
+     *         with the given parameters, or IOStatus.UNSUPPORTED if cloning is
+     *         not supported on this platform
+     */
+    @Override
+    protected int clone(UnixPath src, UnixPath dst, boolean followLinks)
+        throws IOException {
+        int flags = followLinks ? 0 : CLONE_NOFOLLOW;
```

Comment:
Can you put the "{" onto its own line so that it's a bit easier to distinguish the method declaration from the body.

Reply:
Fixed in 2157a1480229fa697c824ae814f11a309627d768.